aptos-move/framework/aptos-framework/sources/vesting.spec.move

```diff
@@ -99,49 +135,111 @@ spec aptos_framework::vesting {
     spec create_vesting_contract {
         // TODO: Data invariant does not hold.
         pragma verify = false;
+        pragma aborts_if_is_partial;
+
+        aborts_if withdrawal_address == @aptos_framework || withdrawal_address == @vm_reserved;
+        aborts_if !exists<account::Account>(withdrawal_address);
+        aborts_if !exists<coin::CoinStore<AptosCoin>>(withdrawal_address);
+        aborts_if len(shareholders) == 0;
+        aborts_if simple_map::spec_len(buy_ins) != len(shareholders);
     }
 
     spec unlock_rewards(contract_address: address) {
         // TODO: Calls `unlock_stake` which is not verified.
+        // Current verification times out even with partial.
```

Comment:
Could you identify which aborts_if conditions lead to timeout?

Reply:
I add comment in the spec, it's mainly from TotalAccumulatedRewardsAbortsIf